### PR TITLE
Added `tabonly` command

### PR DESCRIPTION
**Pull request by osleg**
Executing `tabonly` will close all tabs but active and pinned
Solves #149

### CODE REVIEW

File: src/excmds.ts [hunk 1]
```diff
@@ -594,6 +594,18 @@ async function idFromIndex(index?: number): Promise<number> {
     }
 }
 
+/** Close all tabs but current */
+//#background
+export async function tabonly() {
+    let tabs = browser.tabs.query({pinned: false, active: false})
```
Comment (bovine3dom) [inline src/excmds.ts]: I think you also want `currentWindow: true` in this object.
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/query

Without it, it closes all unpinned tabs across all windows except those which are the 'selected' tab in each window, which isn't what I would expect as a user.

**Comment by bovine3dom** [comment]
Thanks for the PR! Please see the in line comment - the command didn't behave quite as I would expect it to.

**Comment by remofritzsche** [comment]
That's a pretty quick turnaround time for my issue - you guys rock!

File: src/excmds.ts (hunk 1 above)
Comment (osleg) [reply src/excmds.ts]: Oh idd, I didn't even think about multiple FF windows as I olways use just one, will update asap

**Comment by cmcaine** [comment]
Rebased, squashed and slightly modified.

**Comment by cmcaine** [comment]
Merged. Thanks!

**Pull request merged**